### PR TITLE
chore: replace eslint-config-ovh by eslint-config-airbnb-base

### PR DESCRIPTION
**Pull request by antleblanc**
### Description of the Change

cd0c235 — chore: replace eslint-config-ovh by eslint-config-airbnb-base

### Benefits

Aligned  with [Airbnb JavaScript Style Guide](https://github.com/airbnb/javascript).

ref: SCFRCA-2107

### CODE REVIEW

**Comment by antleblanc** [comment]
Previously opened from #580

File: Gruntfile.js [hunk 1]
```diff
@@ -140,9 +140,9 @@ module.exports = function (grunt) {
                 }),
                 dest: "."
             },
-            options: {
-                eslintConfig: require("./.eslintrc.js")
-            }
+            // options: {
```
Comment (marie-j) [inline Gruntfile.js]: Why not simply remove it ?

File: .eslintignore [hunk 2]
```diff
@@ -1 +1,9 @@
-client/app/resources/angular/i18n/
+client/app/account/otrs/filter/html-string-linky.filter.js
```
[the remark this answers is not in the document]
Comment (antleblanc) [reply .eslintignore]: Existing a large amount of errors to these files. That's the reason that I tried to not add too much file on this `.eslintignore` file.

Of course I have to fix them as soon as I can and/or refactor them entirely.

File: .eslintrc [hunk 3]
```diff
@@ -1,34 +1,24 @@
...
         "validator": true,
-        "Raven": true,
-        "RFB": true,
-        "WebUtil": true,
-        "RandExp": true,
-        "EprotectIframeClient": true,
         "zE": true
+    },
+    "rules": {
+        "no-bitwise": ["error", { "allow": ["~"] }]
```
[the remark this answers is not in the document]
Comment (antleblanc) [reply .eslintrc]: @blaryjp is your man:)

**Pull request merged**